evap/staff/templates/staff_semester_view_course.html

```diff
@@ -143,16 +143,16 @@
 {% if state == 'in_evaluation' or state == 'evaluated' or state == 'reviewed' or state == 'published' %}
     {% if not course.is_single_result %}
         <td style="width: 20%" class="multi-progress-bar">
-            {% if course.has_enough_voters_to_publish_grades %}
+            {% if course.can_publish_average_grade %}
                 <span data-toggle="tooltip" data-placement="left" title="{% trans 'Number of voters' %}">
                     {% include 'progress_bar.html' with done=course.num_voters total=course.num_participants icon='user' %}
                 </span>
             {% else %}
-                <span data-toggle="tooltip" data-placement="left" title="{% trans 'Not enough voters to publish results' %}">
+                <span data-toggle="tooltip" data-placement="left" title="{% trans 'Not enough voters to publish average grade' %}">
                     {% include 'progress_bar.html' with done=course.num_voters total=course.num_participants icon='user' warning=True %}
                 </span>
```

Comment:
but there are none if we delete them? we don't even know whether there have been any.

Reply:
but we delete them only on publish, this part here is also shown in other states.

Reply:
this depends mainly on the decision whether we allow staff to see the comments or not. if they can't see them (as currently implemented) i would say they also don't need to know that there are any at all.

Reply:
in case we leave the visibility as it is, the current solution seems inconsistent. showing a zero there as you said or explaining why the user can't see the answers although there are some would both be fine with me.

Reply:
k. will not show them at all then.